lib/travis/api/v3/service.rb

```diff
@@ -142,5 +142,9 @@ def accepted(**payload)
     def not_implemented
       raise NotImplemented
     end
+
+    def com_only_service!
+      raise ComOnlyService unless access_control.private_api? || (ENV['ENV'] || ENV['RACK_ENV']) == 'development'
```

Comment:
I might do ```[ENV['ENV'], ENV['RACK_ENV']].include? 'development'``` or make a method in service that checks for env. I think we do some env checks in other services that could benefit from a method call like ```environment?('development')```. I think the env is being set to Travis.env instead of in the ENV, that's what I found locally, and what the logs and jobs endpoints use when they check the environment. The best would be to add this to our Travis config gem so we could do railsy things like: ```Travis.env.development?```